src/tests/ftest/deployment/basic_checkout.py

```diff
@@ -0,0 +1,101 @@
+        dfuse_mount_dir = self.params.get("mount_dir", "/run/dfuse/*")
+        transfer_block_size = self.params.get("transfer_block_size",
+                                              '/run/ior/iorflags/*')
+        obj_class = self.params.get("obj_class", '/run/ior/iorflags/*')
+        ec_obj_class = self.params.get("ec_oclass", '/run/ior/*')
+        mdtest_params = self.params.get("mdtest_params", "/run/mdtest/*")
+
+
+        #run ior
+        results = self.run_ior_multiple_variants(obj_class, apis, transfer_block_size,
+                                                 flags, dfuse_mount_dir)
+
+        #run ior with different ec oclass
+        results_ec = self.run_ior_multiple_variants(ec_obj_class, [apis[0]],
+                                                    [transfer_block_size[1]],
+                                                    [flags[0]], dfuse_mount_dir)
+        results = results + results_ec
+        self.log.error("Summary of IOR small test results:")
+        errors = False
+        for item in results:
+            self.log.info("  %s  %s", item[0], item[1])
+            if item[0] == "FAIL":
+                errors = True
+        if errors:
+            self.fail("Test FAILED")
+
+        #run mdtest
+        self.run_mdtest_multiple_variants(mdtest_params)
+
+        #run autotest
+        self.log.info("Autotest start")
+        daos_cmd = self.get_daos_command()
+        try:
+            daos_cmd.pool_autotest(pool=self.pool.uuid)
+            self.log.info("daos pool autotest passed.")
+        except CommandFailure as error:
+            self.log.error("Error: %s", error)
+            self.fail("daos pool autotest failed!")
+
+
+class BasicCheckoutDm(DataMoverTestBase):
+    # pylint: disable=too-few-public-methods
+    # pylint: disable=too-many-ancestors
+    """Test Class Description: Test class to wrap datamover test to
+                               run as part of basic checkout and verify
+                               connectivity for lustre FS
+    :avocado: recursive
+    """
+
+    def test_basic_checkout_dm(self):
+        """
+        Test Description: Datamover test to check connection and datamover
+                          functionality with Lustre fs on newly installed
+                          server nodes.
+        :avocado: tags=installation,basiccheckout
+        """
+        # local param
+        dm_ior_options = self.params.get("dm_ior_options", "/run/ior/dm/*")
+
+        # update ior params and run dm test
+        self.ior_cmd.flags.update(dm_ior_options[0])
+        self.ior_cmd.signature.update(dm_ior_options[1])
+        self.ior_cmd.transfer_size.update(dm_ior_options[2])
+        self.ior_cmd.block_size.update(dm_ior_options[3])
+        self.ior_cmd.dfs_dir_oclass.update(dm_ior_options[4])
+        self.ior_cmd.dfs_oclass.update(dm_ior_options[5])
+        self.ior_cmd.test_file.update(dm_ior_options[6])
+        self.ior_cmd.repetitions.update(dm_ior_options[7])
```

Comment:
Maybe hacky, but I wonder if you could set `self.ior_cmd.namespace = "/run/ior/dm/*"` and then call `self.ior_cmd.get_params()`? The benefit would be hat you could add or remove params in the yaml without needing to modify the `.py`. But then maybe the ior/dm should not be under /ior/ in order to avoid conflicts with the non-dm ior options. Thoughts @phender ?

Reply:
Done